src/components/cards/CardUsage.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <h1>Recent Decks using these cards</h1>
+  <div v-if="loading" class="text-center text-gray text-2xl py-8 px-4">
+    <i class="fas fa-circle-notch fa-spin"></i> Loading...
+  </div>
+  <div v-else-if="(!decks || !decks.length)" class="text-center bg-inexhaustible border border-gray-light my-4 py-8">
+    <h2>No Decks found</h2>
+    <p>
+      <button class="btn btn-blue px-4 py-2">Be the First!</button>
```

Comment:
This button doesn't actually do anything; shouldprobably just remove it, since it's actually kind of tricky to create a deck with a card pre-populated in it (which is what the button suggests would happen) given that a Phoenixborn is required before you can start a deck.